tests/validators_test.py

```diff
@@ -271,6 +271,35 @@ def test_units_invalid(units: str):
     assert result.name == 'x'
 
 
+@pytest.mark.parametrize(
+    'units, valid',
+    [
+        ('m', True),
+        ('', True),
```

Comment:
Technically correct. But do we expect nexus files to use an empty unit to indicate `dimensionless`?

Reply:
I don't know, we do have some logic in scippnexus which interestingly forces unrecongized units from nexus fields to scipp `dimensionless`.

https://github.com/scipp/scippnexus/blob/0dbe0d9332333c06ca15f0258527bb37561795d8/src/scippnexus/field.py#L246